changelog.d/templates/entry_title.md.j2

```diff
@@ -0,0 +1 @@
+{{ version if version else "VERSION" }} ({{ date.strftime('%Y-%m-%d') }})
```

Comment:
when would that `"VERSION"` happen? should we just error may be to not end up with some "broken" changelog?

Reply:
With this setup, I don't believe it'll happen; I just copied the template from datalad-next.